guides/updating/update-indy-sdk-to-askar.md

```diff
@@ -20,115 +16,85 @@ Postgres is not supported, yet.
 
 :::caution
 
-The migration script is supported for 0.4.x (or moving from 0.3.x to 0.4.x).
-Please refer to [Migrating from AFJ 0.3.x to 0.4.x](./versions/0.3-to-0.4.md)
-to get to 0.4.x.
+The migration script is supported for 0.4.x (or moving from 0.3.x to 0.4.x). Please refer to [Migrating from AFJ 0.3.x to 0.4.x](./versions/0.3-to-0.4.md) to get to 0.4.x.
```

Comment:
It actually isn't supported for 0.4.0, as the next line says `It is important to note that this script must be ran before you update from 0.3.x to 0.4.x.`